build/source/driver/multi_driver.f90

```diff
@@ -211,23 +212,44 @@ program multi_driver
 integer(i4b)                     :: err=0                      ! error code
 character(len=1024)              :: message=''                 ! error message
 
+! subsetting the model run
+integer(i4b)                     :: strtHRU                    ! index of the starting hru for a subset run
+character(len=10)                :: strTMP                     ! local temp string
```

Comment:
This is a very good question. I am stingy to spend memory thus I tend to use less variables. But I can use the convention after we make the decision on it.